### PR TITLE
Fix padding_mode option in R2Conv

### PR DESCRIPTION
**Pull request by steven-lang**
The condition on the `padding_mode` to check for `zeros` was the wrong way around: If `padding_mode == 'zeros'`, then the default implicit-zero padding of the `Conv2d` module should be used. Else, use a specific padding mode for `torch.nn.functional.pad`.

### CODE REVIEW

**Comment by Gabri95** [comment]
Hi @steven-lang 

Thank you very much for spotting this issue and fixing it!

By the way, thanks for opening our first pull request :)

We are still setting up the instructions and the format for contributions.
You might need to add a 'Signed-off-by' line in your commit to agree with the current license (see [dco](https://github.com/probot/dco)).
I will get back to you once this is ready.

Thanks again!

Gabriele

**Comment by steven-lang** [comment]
Glad I could help! 

Thanks for your awesome work -- incorporating it into my experiments was a very smooth plug-and-play experience. 

I've fixed the commit message and appended the `Signed-off-by` part (I hope the overwrite and force push was ok, else I'd have had to create a new branch and a new PR).
